libs/components/popovers/testing/src/dropdown/harness/dropdown-item-harness.ts

```diff
@@ -0,0 +1,51 @@
+import { HarnessPredicate } from '@angular/cdk/testing';
+import { SkyComponentHarness } from '@skyux/core/testing';
+
+import { SkyDropdownItemHarnessFilters } from './dropdown-item-harness.filters';
+
+export class SkyDropdownItemHarness extends SkyComponentHarness {
+  /**
+   * @internal
+   */
+  public static hostSelector = 'sky-dropdown-item';
+
+  #getItem = this.locatorFor('.sky-dropdown-item');
+  #getButton = this.locatorFor('button,a');
+
+  /**
+   * Gets a `HarnessPredicate` that can be used to search for a
+   * `SkyAutocompleteHarness` that meets certain criteria.
+   */
+  public static with(
+    filters: SkyDropdownItemHarnessFilters
+  ): HarnessPredicate<SkyDropdownItemHarness> {
+    return SkyDropdownItemHarness.getDataSkyIdPredicate(filters)
+      .addOption('textContent', filters.text, async (harness, text) =>
+        HarnessPredicate.stringMatches(await harness.getText(), text)
+      )
+      .addOption('role', filters.ariaRole, async (harness, ariaRole) =>
```

Comment:
The option name and filter property should match:
```
.addOption('ariaRole', filters.ariaRole
.addOption('text', filters.text
```

(Autocomplete harness incorrectly uses `textContent` for `text`; we should fix that later.)

Reply:
oh i thought that was referencing it being called `textContext` if u try and query it directly from the dom lol, which is why i also called it `role` and not `ariaRole`. fixed now!